### ISSUE TITLE
refactor: beautify 404 not found page

### ISSUE THREAD

**Issue by Stefan-Radu** [dothidden/writeups#42]
* we can do something fun for this like an md5 hash / base64 text / or just some leet speak like `n07 f0uNd`
* make it big so it fills the screen and maybe ranbow? =)))))))

**Comment by Mathis-Dory** [comment]
I agree we can do something fun, if someone has some time don't hesitate to claim the issue :)

**Comment by Stefan-Radu** [comment]
> I agree we can do something fun, if someone has some time don't hesitate to claim the issue :)

I would try something, but idk when xD

**Comment by Costinteo** [comment]
implement pong on 404 page
make 4s be paddles and 0 the ball
